src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,89 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+import React from 'react';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const numberOfPages: number = Math.ceil(total / perPage);
+
+  return (
+    <ul className="pagination">
+      <li
+        className={cn('page-item', {
+          disabled: currentPage === 1,
+          active: currentPage === 1,
+        })}
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1}
+          onClick={() => {
+            if (currentPage !== 1) {
+              onPageChange(currentPage - 1);
+            }
+          }}
+        >
+          «
+        </a>
+      </li>
+
+      {Array(numberOfPages)
+        .fill(null)
+        .map((_, index) => {
+          return (
+            <li
+              key={index + 1}
+              className={cn('page-item', {
+                active: currentPage === index + 1,
+              })}
+            >
+              <a
+                data-cy="pageLink"
+                className="page-link"
+                href="#1"
```

Comment:
The `href` attribute here is hardcoded to `#1`, which doesn't seem to be dynamic. You should ensure that the `href` reflects the actual page that will be navigated to when the link is clicked. However, if you are preventing the default action and using onClick to handle the page change, you might not need the `href` attribute at all, or you could use `href='#!'` to indicate that it's a placeholder for a dynamic link.